### PR TITLE
Add bazel requirement and MacOS info to docs, add --rm to bazel-container tool

### PR DESCRIPTION
**Pull request by oliver-amzn**
This change is [Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/74)

### CODE REVIEW

File: tools/bazel-container [hunk 1]
```diff
@@ -49,6 +49,7 @@ main() {
 
   local -a docker_options=(
     '-it'
+    '--rm'
```
Comment (oliver-amzn) [inline tools/bazel-container:52]: This was added before here: https://github.com/world-federation-of-advertisers/experimental/pull/87
but got lost

Comment (corbantek) [reply tools/bazel-container:52]: Yeah I had added that locally myself.  Good call.

**Review by corbantek — APPROVED** [review]
LGTM

**Review by efoxepstein — APPROVED** [review]

File: tools/bazel-container (hunk 1 above)
Comment (oliver-amzn) [reply tools/bazel-container:52]: Anything else you added locally that you think is worth adding here?

**Comment by oliver-amzn** [comment]
From the GH Actions linter log: `Error: ReserveCacheError: Unable to reserve cache with key ktlint-0.40.0, another job may be creating this cache.` @SanjayVas - looks like [this block here](https://github.com/world-federation-of-advertisers/actions/blob/fb2854567159e31688b35bdc4587223ae46afeb3/setup-linters/index.js#L52-L54) isn't working as expected and outputting a warning but making the GH Action fail?

**Review by efoxepstein — APPROVED** [review]
> Reviewed 2 of 2 files at r1.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/74)* status: all files reviewed, 1 unresolved discussion (waiting on @corbantek and @SanjayVas)

**Review by SanjayVas — APPROVED** [review]
It's fixed by world-federation-of-advertisers/actions#12, which is in the v1.4.1 release. Updating the workflow to use the floating v1 release should do the trick.

> Reviewed 2 of 2 files at r1.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/74#-:-MaUrZa701r_Aan4-EaV:b87np85)* status: all files reviewed, 1 unresolved discussion (waiting on @corbantek)

**Comment by oliver-amzn** [comment]
Fixed the lint issue by pulling in v1 of the actions - does this need to get re-reviewed via Reviewable or is it good to merge?

**Review by SanjayVas — APPROVED** [review]
> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/74)* status: all files reviewed, 1 unresolved discussion (waiting on @corbantek)

**Review by SanjayVas — APPROVED** [review]
I had already approved with the assumption that you'd resolve pending comments, so you were good to go. I've reapproved to even further give that stamp.

> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/74#-:-MaVuaCR1XrP6-8rQfHN:bog8o9d)* status: all files reviewed, 1 unresolved discussion (waiting on @corbantek)

**Review by corbantek — APPROVED** [review]

**Pull request merged**